client/my-sites/upgrades/domain-management/email/google-apps-user-item.jsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+
+const GoogleAppsUserItem = React.createClass( {
+	propTypes: {
+		user: React.PropTypes.object.isRequired,
+		onClick: React.PropTypes.func
+	},
+
+	shouldComponentUpdate( nextProps ) {
+		return this.props.user !== nextProps.user || this.props.onClick !== nextProps.onClick;
+	},
+
+	render() {
+		return (
+			<li>
+				<span className="google-apps-users-card__user-email">
```

Comment:
Not that I know of. But just to clarify - what would need to be cleaned up in this case?